File: SDKs/Aspose.Words_Cloud_SDK_for_CSharp/src/Com/Aspose/Words/Model/DocumentStatData.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class DocumentStatData {
    public int WordCount { get; set; }

    public int ParagraphCount { get; set; }

    public int PageCount { get; set; }

    public List<PageStatData> PageStatData { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DocumentStatData {\n");
      sb.Append("  WordCount: ").Append(WordCount).Append("\n");
      sb.Append("  ParagraphCount: ").Append(ParagraphCount).Append("\n");
      sb.Append("  PageCount: ").Append(PageCount).Append("\n");
      sb.Append("  PageStatData: ").Append(PageStatData).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/DocumentStatData.java
package com.aspose.words.model;

import java.util.*;
public class DocumentStatData {
  private Integer WordCount = null;
  private Integer ParagraphCount = null;
  private Integer PageCount = null;
  private List<PageStatData> PageStatData = new ArrayList<PageStatData>();
  /**
	 * getWordCount
	 * Gets Integer
	 * @return WordCount
	 */
  public Integer getWordCount() {
    return WordCount;
  }

	/**
	 * setWordCount
	 * Sets Integer
	 * @param WordCount Integer
	 */
  public void setWordCount(Integer WordCount) {
    this.WordCount = WordCount;
  }

  /**
	 * getParagraphCount
	 * Gets Integer
	 * @return ParagraphCount
	 */
  public Integer getParagraphCount() {
    return ParagraphCount;
  }

	/**
	 * setParagraphCount
	 * Sets Integer
	 * @param ParagraphCount Integer
	 */
  public void setParagraphCount(Integer ParagraphCount) {
    this.ParagraphCount = ParagraphCount;
  }

  /**
	 * getPageCount
	 * Gets Integer
	 * @return PageCount
	 */
  public Integer getPageCount() {
    return PageCount;
  }

	/**
	 * setPageCount
	 * Sets Integer
	 * @param PageCount Integer
	 */
  public void setPageCount(Integer PageCount) {
    this.PageCount = PageCount;
  }

  /**
	 * getPageStatData
	 * Gets List<PageStatData>
	 * @return PageStatData
	 */
  public List<PageStatData> getPageStatData() {
    return PageStatData;
  }

	/**
	 * setPageStatData
	 * Sets List<PageStatData>
	 * @param PageStatData List<PageStatData>
	 */
  public void setPageStatData(List<PageStatData> PageStatData) {
    this.PageStatData = PageStatData;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class DocumentStatData {\n");
    sb.append("  WordCount: ").append(WordCount).append("\n");
    sb.append("  ParagraphCount: ").append(ParagraphCount).append("\n");
    sb.append("  PageCount: ").append(PageCount).append("\n");
    sb.append("  PageStatData: ").append(PageStatData).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

